it/shared/auth/AuthMainAgentsOnlyISpec.scala

```diff
@@ -59,7 +61,7 @@ abstract class AuthMainAgentsOnlyISpec extends IntegrationBaseSpec {
         }
 
         val response: WSResponse = sendMtdRequest(request())
-        response.status shouldBe expectedMtdSuccessStatus
+        (response.status == expectedMtdSuccessStatus || response.status == alternativeExpectedMtdSuccessStatus) shouldBe true
```

Comment:
We can chang the expectedMtdSuccessStatus value to 204 is this is what the test now expects. In this instance the test makes a request to the /ignore endpoint which now returns 204